x/lockup/keeper/lock.go

```diff
@@ -309,6 +309,13 @@ func (k Keeper) ClearAllLockRefKeys(ctx sdk.Context) {
 	k.clearLockRefKeysByPrefix(ctx, types.KeyPrefixUnlocking)
 }
 
+func (k Keeper) ClearAllAccumulationStores(ctx sdk.Context) {
+  denoms := k.getAccumulationDenoms(ctx)
```

Comment:
Can we combine the two methods, and just delete everything under the subspace `prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefixLockAccumulation)` ?